sale_disable_inventory_check/models/product_category.py

```diff
@@ -0,0 +1,16 @@
+# Copyright Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    check_stock_on_sale = fields.Boolean(
+        company_dependent=True,
+        help="Uncheck if you want to disable warning 'Not enough inventory'"
+             " when there isn't enough product in stock",
+        string="Disable Check Stock Message",
```

Comment:
The `help` and `string` contradict each other: `help` says you should "Uncheck to disable the warning" -> If true, do check. But `string` says "Disable Check Stock message" -> If true, do not check.

I think the `string` should be changed to be "Check Stock on Sale".